.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js:
+- 4.1.1
+before_script:
+- gem install percy-cli
+after_success:
+- '[ "$TRAVIS_SECURE_ENV_VARS" = "true" ] && npm run perceptual-diff'
+sudo: false
+deploy:
+  provider: npm
+  email: d2ltravisdeploy@d2l.com
+  api_key:
```

Comment:
The percy stuff will likely fail until we get our license upgraded (in progress)